tests/Unit/AuditPageTest.php

```
@@ -40,6 +40,55 @@ public function test_register_menu_skips_audit_and_ai_discovery_without_capabili
+
+    public function test_selected_month_falls_back_to_current_wordpress_month(): void
+    {
+        Functions\expect('current_datetime')->once()->andReturn(new \DateTimeImmutable('2026-08-16'));
+        $_GET['month'] = '2026-13';
+
+        try {
+            $this->assertSame('2026-08', self::callPrivate(AuditPage::class, 'selected_month'));
+        } finally {
+            unset($_GET['month']);
+        }
+    }
```

PHPMD (2.15.0)

[error] test_selected_month_falls_back_to_current_wordpress_month accesses the super-global variable $_GET. (undefined)

(Superglobals)